timeseries/src/main/java/com/hurence/logisland/timeseries/sax/SaxAnalyzer.java

```diff
@@ -0,0 +1,75 @@
+package com.hurence.logisland.timeseries.sax;
+
+import com.sun.javafx.collections.MappingChange;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.*;
+
+
+public class SaxAnalyzer {
+
+    public static Map<Integer, Integer> countOccurrence(int[] numbersToProcess) {
+        int[] possibleNumbers = new int[30];
+        Map<Integer, Integer> result = new HashMap<Integer, Integer>();
+
+        for (int i = 0; i < numbersToProcess.length; ++i) {
+            possibleNumbers[numbersToProcess[i]] = possibleNumbers[numbersToProcess[i]] + 1;
+            result.put(numbersToProcess[i], possibleNumbers[numbersToProcess[i]]);
+        }
+
+        return result;
+    }
+    public static List<Integer> saxThreshold(String str){
+        double l = str.length();
+        //Transform our string into an array of characters
+        char[] strArray = new char[str.length()];
+        for (int i = 0; i < str.length(); i++)
+            strArray[i] = str.charAt(i);
+        //Array of distances between characters
+        int[] buff = new int[str.length()-1];
+        for (int j =0; j< str.length()-1; j++)
+            buff[j] = Math.abs((int)(strArray [j+1] - strArray[j]));
+
+        //Counting occurrences
```

Comment:
This function should probably be separate one to be reused if reusable ;) may be also the part that computes the array of distances ?